kyaml/fn/runtime/exec/exec.go

```diff
@@ -28,9 +32,22 @@ func (c *Filter) Filter(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
 }
 
 func (c *Filter) Run(reader io.Reader, writer io.Writer) error {
+	var workingDir string
+	var err error
+
+	if c.WorkingDir != "" {
+		workingDir = c.WorkingDir
+	} else {
+		workingDir, err = os.Getwd()
```

Comment:
As a note, this is needed because the container runtime makes a call to `docker` using this Filter, in which case c.WorkingDir is empty and it should run in the current directory.